deltachat-ios/Controller/AddGroupMembersViewController.swift

```diff
@@ -99,20 +107,13 @@ class AddGroupMembersViewController: GroupMembersViewController {
     }
 
     override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        switch indexPath.section {
-        case sectionNewContact:
-            return Constants.defaultCellHeight
-        case sectionMemberList:
-            return ContactCell.cellHeight
-        default:
-            return Constants.defaultCellHeight
-        }
+        return indexPath.section == sectionMemberList ? ContactCell.cellHeight : UITableView.automaticDimension
```

Comment:
generally all ViewControllers should avoid `tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath)` if we want to have self-sizing cells. The only exception for now is ContactCell. Returning `UITableView.automaticDimension` indicates that iOS will respect the self-sizing of cells